### PR TITLE
bpo-42392: Remove loop parameter from asyncio.tasks and asyncio.subprocess

### PR DESCRIPTION
**Pull request by uriyyo**
Remove loop parameter from `asyncio.tasks` and `asyncio.subprocess`.

This is a final PR regarding removing the deprecated `loop` parameter.

https://bugs.python.org/issue42392

### CODE REVIEW

File: Lib/asyncio/subprocess.py [hunk 1]
```diff
@@ -221,18 +211,12 @@ async def create_subprocess_shell(cmd, stdin=None, stdout=None, stderr=None,
...
-                      "and scheduled for removal in Python 3.10.",
-                      DeprecationWarning,
-                      stacklevel=2
-        )
+                                 stderr=None, limit=streams._DEFAULT_LIMIT,
+                                 **kwds):
+    loop = events.get_running_loop()
     protocol_factory = lambda: SubprocessStreamProtocol(limit=limit,
                                                         loop=loop)
+
```
Comment (jstasiak) [inline Lib/asyncio/subprocess.py]: Unnecessary whitespace change?

File: Lib/asyncio/tasks.py [hunk 2]
```diff
@@ -747,13 +726,9 @@ def gather(*coros_or_futures, loop=None, return_exceptions=False):
     after catching an exception (raised by one of the awaitables) from
     gather won't cancel any other awaitables.
     """
+
```
Comment (jstasiak) [inline Lib/asyncio/tasks.py]: Another unnecessary addition.

**Review by jstasiak — APPROVED** [review]
LGTM

File: Lib/asyncio/subprocess.py (hunk 1 above)
Comment (uriyyo) [reply Lib/asyncio/subprocess.py]: Yup, will remove it, thanks

File: Lib/asyncio/tasks.py (hunk 2 above)
Comment (uriyyo) [reply Lib/asyncio/tasks.py]: Yup, will remove it, thanks

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!
